frontend/src/components/pages/rp-connect/Pipelines.Create.tsx

```diff
@@ -20,6 +20,7 @@ import PipelinesYamlEditor from '../../misc/PipelinesYamlEditor';
 import { pipelinesApi } from '../../../state/backendApi';
 import { DefaultSkeleton } from '../../../utils/tsxUtils';
 import { Link } from 'react-router-dom';
+import { Link as ChLink } from '@redpanda-data/ui';
```

Comment:
nit (no need to change this now): I think we should follow some convention through the code when importing Links from `react-router-dom` and `redpanda-data`. 

On majority of pages I saw the convention that `Link` from `redpanda-data` is imported as `Link` and `Link` from `react-router-dom` as `ReactRouterLink`. I think following the same convention would make the code more clean.